website/sidebars.json

```diff
@@ -76,6 +76,10 @@
       "reference_hyperview_component",
       "reference_custom_elements",
       "reference_custom_behaviors"
+    ],
+    "Hyperview Navigation":[
```

Comment:
For some reason, this section is not showing up in the sidebar on the preview site.

Reply:
Should be working now. You may need to run `yarn build` to see the updates.